src/components/layout/nav/MainNavMenu.tsx

```diff
@@ -69,6 +71,9 @@ export default function MainNavMenu({ children }: { children?: React.ReactNode }
           {t('nav.blog')}
         </Button>
       </Grid>
+      <Grid item>
+        <DonationMenu />
+      </Grid>
```

Comment:
Just confirmed with the design team that the best UX practices are to have the Blog item next to the most right position, just before the locale.